Comanche055/CONIC_SUBROUTINES.agc

```diff
@@ -111,7 +111,7 @@
 #							MAJOR AXIS), FOR SPEED OF CONVERGENCE, BUT IF EITHER CONDITION FAILS, XKEPNEW IS RESET
 #							BY KEPLER TO A POOR BUT VALID GUESS.
 #	TC		+28			DP PREV. VALUE OF TIME IN CENTISECS. MUST BE LESS THAN ONE ORBITAL PERIOD.
-#	XPREV		+17 FOR EARTH		PRVIOUS VALUE OF X IN SQRT(METERS).  MUST BE LESS THAN AN X CORRESPONDING TO ONE
```

Comment:
Nope. Correct is DP PREV. http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1264.jpg